hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/acl/OMBucketAddAclRequest.java

```diff
@@ -122,8 +122,7 @@ OMClientResponse onSuccess(OMResponse.Builder omResponse,
   @Override
   OMClientResponse onFailure(OMResponse.Builder omResponse,
```

Comment:
You can just remove this onFailure implementation in each sub class.

Reply:
Thanks for review,I will delete it in each class

Reply:
Hi, @ChenSammi I have fixed it, PTAL! Thanks!